### DISCUSSION TITLE
No longer works

### DISCUSSION THREAD

**Discussion by AddisonPrairie** [llm-workflow-engine/llm-workflow-engine#79] [Q&A] [answered]
chatgpt-wrapper no longer works for me, even when I can log into chatGPT and use chatGPT manually, every response I get through chatgpt-wrapper is just "There was an error generating a response"

**Comment by Tecuya** [comment, marked as answer]
I just pushed an update.  It seems they changed the name of the model in the request object.  After changing it I was able to get chatgpt-wrapper working again.  Please update!  Thanks for the report

**Reply by AddisonPrairie** [reply]
It seems to be working now, thanks!
